Azure/sonic-utilities
# [db_migrate] Add config db mux keys migrator

#### What I did
To migrate the mux-related key name changes introduced by PR: https://github.com/Azure/sonic-swss/pull/2135

Signed-off-by: Longxiang Lyu <lolv@microsoft.com>

#### How I did it
1. add key name migrator functions to script `db_migrate.py`
2. add unit test

#### How to verify it
1. check the config db after a `config reload`:
```
root@vlab-05:~# redis-cli -n 4
127.0.0.1:6379[4]> keys *LINK*
 1) "LINKMGR_CABLE|Ethernet60"
 2) "LINKMGR_CABLE|Ethernet4"
 3) "LINKMGR_CABLE|Ethernet12"
 4) "LINKMGR_CABLE|Ethernet8"
 5) "LINKMGR_CABLE|Ethernet28"
 6) "LINKMGR_CABLE|Ethernet56"
 7) "LINKMGR_CABLE|Ethernet36"
 8) "LINKMGR_CABLE|Ethernet80"
 9) "LINKMGR_CABLE|Ethernet68"
10) "LINKMGR_CABLE|Ethernet52"
11) "LINKMGR_CABLE|Ethernet64"
12) "LINKMGR_CABLE|Ethernet44"
13) "LINKMGR_CABLE|Ethernet96"
14) "LINK_MANAGER|LINK_PROBER"
15) "LINKMGR_CABLE|Ethernet20"
16) "LINKMGR_CABLE|Ethernet24"
17) "LINKMGR_CABLE|Ethernet48"
18) "LINKMGR_CABLE|Ethernet40"
19) "LINKMGR_CABLE|Ethernet92"
20) "LINKMGR_CABLE|Ethernet32"
21) "LINKMGR_CABLE|Ethernet84"
22) "LINKMGR_CABLE|Ethernet72"
23) "LINKMGR_CABLE|Ethernet88"
24) "LINKMGR_CABLE|Ethernet76"
25) "LINKMGR_CABLE|Ethernet16"
```

#### Previous command output (if the output of a command-line utility has changed)

#### New command output (if the output of a command-line utility has changed)
